src/client/components/styled-components/AddOrganizationError.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const AddOrganizationError = props => (
```

Comment:
Let's rename to something more generic, like `ErrorIcon`

Reply:
No problem.